proto/cosmos/upgrade/v1beta1/query.proto

```diff
@@ -26,6 +26,11 @@ service Query {
   rpc UpgradedConsensusState(QueryUpgradedConsensusStateRequest) returns (QueryUpgradedConsensusStateResponse) {
     option (google.api.http).get = "/cosmos/upgrade/v1beta1/upgraded_consensus_state/{last_height}";
   }
+
+  // VersionMap queries the version map from state.
+  rpc VersionMap(QueryVersionMapRequest) returns (QueryVersionMapResponse) {
```

Comment:
Since we're not returning a map anymore, should we call this `rpc ModuleVersions`

Reply:
[renamed rpc method](https://github.com/cosmos/cosmos-sdk/pull/9073/commits/65c8de5455cb079853597fab3e24db8dff1ae365#diff-181d4d746fed56f6aa36d30359c69fe9e04c6c638aa885abd06fe33cae02c934R31)